pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -192,7 +192,7 @@ type KubeletConfiguration struct {
 	//   "promiscuous-bridge": make the container bridge promiscuous.
 	//   "hairpin-veth":       set the hairpin flag on container veth interfaces.
 	//   "none":               do nothing.
-	// Generally, one must set --hairpin-mode=veth-flag to achieve hairpin NAT,
+	// Generally, one must set --hairpin-mode=hairpin-veth to achieve hairpin NAT,
```

Comment:
can you also update the comment in `v1alpha1/types.go`?

Reply:
nice point! It's fixed now. PTAL. THANKS!~